frontend/src/composables/useShootStatusCredentialRotation.js

```diff
@@ -73,22 +66,11 @@ export const twoStepRotationTypes = Object.freeze(filter(rotationTypes, {
   twoStep: true,
 }))
 
-const shootPropertyMappings = Object.freeze({
-  shootEnableStaticTokenKubeconfig: 'spec.kubernetes.enableStaticTokenKubeconfig',
-  shootCredentialsRotation: ['status.credentials.rotation', {}],
-})
-
 export function useShootStatusCredentialRotation (state, options = {}) {
-  const {
-    shootEnableStaticTokenKubeconfig,
-    shootCredentialsRotation,
-  } = mapValues(shootPropertyMappings, args => {
-    const [path, defaultValue] = Array.isArray(args) ? args : [args]
-    return computed(() => get(state.value, path, defaultValue))
-  })
-
   options.type ??= 'ALL_CREDENTIALS'
 
+  const shootCredentialsRotation = computed(() => get(state.value, 'status.credentials.rotation', {}))
```

Comment:
This should be a linter error? I will find out why not.
```suggestion
  const shootCredentialsRotation = computed(() => get(state.value, ['status', 'credentials', 'rotation'], {}))
```